### PR TITLE
LPS-113548 Move zIndex definitions to frontend-js-web

### PR DESCRIPTION
**Pull request by kresimir-coko**
Pretty simple, just moving a static object over to a different module. Tested in DXP via the developer console and it works as it should.

### CODE REVIEW

**Comment by kresimir-coko** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/zIndex.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default {
```
Comment (bryceosterhaus) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/zIndex.js:15]: This is gonna be nice to have this in its own file for future searching. I always forget where we defined these when they were in aui-web

**Review by bryceosterhaus — APPROVED** [review]
LGTM!

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/zIndex.js (hunk 1 above)
Comment (kresimir-coko) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/zIndex.js:15]: 😉

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Comment by kresimir-coko** [comment]
ci:forward

**Pull request closed without merging**